en_US/data-integration/rules.md

```diff
@@ -30,72 +52,98 @@ WHERE
+
 ::: tip
-The console output action is only used for debugging. If it is used in the production environment, it will cause performance degradation.
+The console output action should only be used for debugging. If it is used in the production environment, it may cause performance problems.
 :::
 
-## Typical Use Cases of Rules
+#### Forwarding with Data Bridge
+
+If select **Forwarding with Data Bridge**, the target message will be republished to the selected data bridge. All you need is to select the target Data bridge from the Data bridge drop-down list. 
+
+For more information on how to create a data bridge, see [Data bridge](../data-integration/data-bridges.md).
+
+## Typical Use Cases
+
+### Action Monitoring
+
+Rules can be used to monitor the status of clients connected to the broker. 
+
+For example, if a client representing a door in a smart home is disconnected, a rule can be used to send a notification to a dashboard for the smart home. This helps ensure that important events are promptly acted upon.
+
+### Data Filtering
+
+In certain scenarios, only specific types of data are relevant, and processing all incoming data may not be necessary. 
+
+For example, in truck fleet management, vehicle sensors collect and report a large amount of operation data. However, the application platform may only be interested in data when the vehicle speed is greater than 40 km/h. 
+
+In this case, rules can be used to filter messages conditionally and only write data that meets the conditions to a message queue. This can help optimize system performance and reduce storage requirements.
+
+### Message Routing
+
+Rules can also be used to route messages to different topics based on the content of the message. This allows for more fine-grained control over message distribution. 
+
+For example, a rule can be used to route messages containing a temperature greater than 100 degrees to a topic for high-temperature alarms. This can help ensure that messages are delivered to the appropriate destination for further processing.
 
-- Action monitoring: in the development of smart home smart door locks, the door locks will be offline due to network, power failure, man-made damage and other reasons, resulting in abnormal functions. Use rule configuration to monitor offline events and push the fault information to the application service, so as to realize the ability of fault detection at the access layer at the first time;
+### Message Encoding and Decoding
 
-- Data filtering: in the truck fleet management of the Internet of vehicles, vehicle sensors collect and report a large amount of operation data. The application platform only focuses on the data when the vehicle speed is greater than 40 km/h. In this scenario, rules can be used to filter messages conditionally and write data that meets the conditions to the business message queue;
+For cases where the format transform of messages is needed, rules can be used to encode and decode messages as required.
```

Comment:
the format transform of messages is needed -> the message format needs to be changes